app/controllers/concerns/document_attachable.rb

```diff
@@ -25,6 +25,10 @@ def document_provided?
     tribunal_case&.documents(document_key)&.any? || document.present?
   end
 
+  def document_presence
```

Comment:
Yes, I was unsure about this, but it seems to be the recommended naming, at least according to Rails documentation:

<img width="665" alt="screen shot 2017-07-12 at 09 54 30" src="https://user-images.githubusercontent.com/687910/28109807-458eab16-66e8-11e7-960b-8da86ceaccb6.png">

Or we can name it to something a bit more verbose, along the lines: `document_cannot_be_blank` if we follow this other example:

<img width="655" alt="screen shot 2017-07-12 at 09 58 54" src="https://user-images.githubusercontent.com/687910/28109957-c1e50bba-66e8-11e7-828e-c8f752d33cbb.png">